### ISSUE TITLE
Middle-click doesn't work on editable fields

### ISSUE THREAD

**Issue by simonbcn** [CanisLupus/swift-selection-search#110]
```
Swift Selection Search v3.24.0
Firefox v61.0.1
Arch Linux
```

I've set the `Opening behaviour` to middle-mouse button and enabled `Allow popup on editable fields`. But it doesn't work on editable fields. 
I have tried changing the value of `middlemouse.paste` in `about:config` but it doesn't work.

**Comment by CanisLupus** [comment]
You are right! Something is wrong here... it works sometimes, but after deselecting the text in the input field it doesn't. Then it works again if I select normal text first. I'll try to look into this soon.

Thanks for reporting! :)

Cheers,
Daniel

**Comment by CanisLupus** [comment]
I just submitted version 3.25.1 to the add-ons market with a _supposed_ fix for this problem.

When this issue happened to me it seemed to be catching a selection with 0 width somewhere in the page that was not the input field itself. As the code prioritized "non-editable field selections" it refused to appear for the input field while that selection existed. I changed it so it checks for editable fields regardless.

Please tell me if the problem persists for you, since I'm not sure that this covers the situation you've been experiencing. :)

Thanks!
Daniel

**Comment by simonbcn** [comment]
It works now. Thanks. :+1:

**Comment by simonbcn** [comment]
With the last update it fails again.
```
Firefox 67.0.4
Swift Selection Search v3.31.1
```
My options: 
[SSS settings backup (2019-06-27T09.53.09.483Z).zip](https://github.com/CanisLupus/swift-selection-search/files/3334025/SSS.settings.backup.2019-06-27T09.53.09.483Z.zip)

**Comment by CanisLupus** [comment]
Oops, thank you for reporting this. I've checked and this problem was caused by changes I made in version 3.31.0 to fix issue #141.

I've fixed the problem just now but it's not in the addons site yet. This update will unfortunately have to wait at least a few days. I'll explain.

Mozilla is changing their review process for featured extensions and SSS is one of them. They began requiring a detailed review this month.

SSS update 3.32.0 (which does NOT contain the fix I just made) is currently in the review queue and might take a few days. When it finishes, I will be able to submit a new version 3.32.1 that contains the fix. Hopefully that second review will take less time, but it's up to them.

Sorry for this. I'll leave this issue open until the fix reaches the addons site.

Cheers!
Daniel
